test/cpp/c10d/TCPStoreTest.cpp

```diff
@@ -16,6 +16,7 @@ constexpr int64_t kShortStoreTimeoutMillis = 100;
 constexpr int defaultTimeout = 20;
 
 c10::intrusive_ptr<c10d::TCPStore> _createServer(
+    bool useLibUV,
```

Comment:
curious, do you know which test it was? Was it one of the added ones

Reply:
the issue was that the test passed port 0 and we asserted the the allocated port matched that, which will never work